### PR TITLE
TELCODOCS-236: latest RN updates

### PR DESCRIPTION
**Pull request by jcfrye77**
[TELCODOCS-236](https://issues.redhat.com/browse/TELCODOCS-236): updates to RN to include [https://bugzilla.redhat.com/show_bug.cgi?id=1949859](url) and update [https://bugzilla.redhat.com/show_bug.cgi?id=1972213](url) BZ#.

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -1368,7 +1368,7 @@ The `authentication` and `openshift-apiserver` Operators now ignore the `oauth-a
...
 
-* Previously, in UEFI mode, the `ironic-python-agent` created a UEFI bootloader entry after downloading the RHCOS image. When using an RHCOS image based on RHEL 8.4, the image could fail to boot using this entry. If the entry installed by Ironic was used when booting the image, the boot could fail and output a BIOS error screen. This is fixed by the `ironic-python-agent` configuring the boot entry based on a CSV file located in the image, instead of using a fixed boot entry. The image boots properly without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966129[*BZ#1966129*])
+* Previously, in UEFI mode, the `ironic-python-agent` created a UEFI bootloader entry after downloading the RHCOS image. When using an RHCOS image based on RHEL 8.4, the image could fail to boot using this entry. If the entry installed by Ironic was used when booting the image, the boot could fail and output a BIOS error screen. This is fixed by the `ironic-python-agent` configuring the boot entry based on a CSV file located in the image, instead of using a fixed boot entry. The image boots properly without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972213[*BZ#1972213*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-8-release-notes.adoc:1371]: Are you sure that this BZ link needs to change?

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -1609,6 +1609,8 @@ FATAL failed to fetch Metadata: failed to load asset "Install Config": platform.
 
 * Previously, the Azure clusters were created with the disk type of Premium_LRS and with an instance type that did not support PremiumIO capabilities which caused the cluster to fail. This update checks to see if the instance type picked has the PremiumIO capabilities only if the disk type is Premium_LRS which is the default disk type. The code queries the Azure subscription and region to get the information required and returns an error if the conditions are not met. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1931115[*BZ#1931115*])
 
+* Previously, the API VIP could become unavailable on the bootstrap when the API server restarts, which makes the provisioning services unavailable and provisioning fails. The provisioning services (Ironic) are now included in the VIP health checks, and the API VIP remains available. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[*BZ#1949859*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-8-release-notes.adoc:1612]:
```suggestion
* Previously, the API VIP could become unavailable on the bootstrap when the API server restarted, which made the provisioning services unavailable and caused provisioning to fail. The provisioning services (Ironic) are now included in the VIP health checks, and the API VIP remains available. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[*BZ#1949859*])
```

**Review by kalexand-rh — COMMENTED** [review]
I have a question and some tense changes.

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (jcfrye77) [reply release_notes/ocp-4-8-release-notes.adoc:1371]: Yes, I got confirmation from the assignee that BZ#1966129 was incorrect (that is a clone). It needed to be updated to BZ#1972213.

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (jcfrye77) [reply release_notes/ocp-4-8-release-notes.adoc:1612]: That's a great catch, I'm sorry I did not fix that previously. I will make that update. Thanks very much.

**Comment by jcfrye77** [comment]
@kalexand-rh Thank you for the quick and helpful review! I will make the suggested corrections. Also, if we could please hold this open until Tuesday (just in case any more changes come through). Thank you again for all your time and help.

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (kalexand-rh) [reply release_notes/ocp-4-8-release-notes.adoc:1371]: Thanks! The first bug sounded more like the issue that was being addressed here. :)

**Comment by kalexand-rh** [comment]
> @kalexand-rh Thank you for the quick and helpful review! I will make the suggested corrections. Also, if we could please hold this open until Tuesday (just in case any more changes come through). Thank you again for all your time and help.

Sounds good, @jcfrye77! Send another message when you're ready for this PR to be merged.

**Comment by jcfrye77** [comment]
@kalexand-rh We had another late release note come in, which I've incorporated just now. When you have a chance, I'd appreciate a review of the latest so I can make changes before the upcoming deadline. Thank you again!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 3]
```diff
@@ -1675,6 +1677,8 @@ FATAL failed to fetch Metadata: failed to load asset "Install Config": platform.
 
 * Previously, and for unknown reasons, a kubelet could register the wrong IP address for a node. As a consequence, the node would be in a `NotReady` state until it was rebooted. Now, the systemd manager configuration is reloaded with the valid IP address as an environment variable, meaning that nodes no longer enter a `NotReady` state because a kubelet registered the wrong IP address. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1940939[*BZ#1940939*])
 
+* Previously, refactoring for a shadowed variable caused a regression related to the use of the checkpoint file, and SR-IOV pod sandboxes would not start up. A check for the path of the kubelet socket was not properly accounted for during the refactor. The fix properly restores the check for the kubelet socket path, and now the SR-IOV pod sandboxes are properly created. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1968625[*BZ#1968625*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-8-release-notes.adoc:1680]: > 🚫  start up verb
Do not use. Use "start"

**Review by kalexand-rh — COMMENTED** [review]
I have one style pick. This change otherwise looks good!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 3 above)
Comment (jcfrye77) [reply release_notes/ocp-4-8-release-notes.adoc:1680]: That's a very helpful catch, thank you! I appreciate the quick review.

**Comment by jcfrye77** [comment]
@kalexand-rh I've made the correction. The only other new change was that one more BZ number/link needed to be changed (I had misunderstood and the original link was to the cloned bug). 
No new release notes or updates have come in today, so this is finally ready to be merged. Thank you again for all your help and patience!

**Pull request merged**